tests/plugins/test_store_metadata.py

```diff
@@ -133,23 +132,17 @@ def test_metadata_plugin(tmpdir):
     workflow.builder.df_path = df.dockerfile_path
     workflow.builder.df_dir = str(tmpdir)
 
-    workflow.prebuild_results = {
-        DistgitFetchArtefactsPlugin.key: "artefact1\nartefact2",
-    }
+    workflow.prebuild_results = {}
     workflow.postbuild_results = {
         PostBuildRPMqaPlugin.key: "rpm1\nrpm2",
     }
     workflow.plugins_timestamps = {
-        DistgitFetchArtefactsPlugin.key: (initial_timestamp + timedelta(seconds=1)).isoformat(),
         PostBuildRPMqaPlugin.key: (initial_timestamp + timedelta(seconds=3)).isoformat(),
     }
     workflow.plugins_durations = {
-        DistgitFetchArtefactsPlugin.key: 2.02,
         PostBuildRPMqaPlugin.key: 3.03,
     }
-    workflow.plugins_errors = {
-        DistgitFetchArtefactsPlugin.key: 'foo'
-    }
+    workflow.plugins_errors = {}
```

Comment:
I'd rather leave this for additional clarity what plugins are running. This will also be useful for further changes